app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
```

Comment:
Like model must have a method that updates the likes counter for a post as it's a project requirement. Kindly implement that 👍